README.md

```diff
@@ -14,17 +14,18 @@ Our Android template: **[template](https://github.com/nimblehq/android-templates
 ## Setup
 
 1. Clone or download this repository to your local machine, then extract and open the folder
-2. Install [Kscript](https://github.com/holgerbrandl/kscript#installation)
-3. Run `cd scripts` to get into the scripts directory
-4. Run `kscript new_project.kts` to create a new project with the following arguments:
+2. Install [Kotlin](https://github.com/JetBrains/kotlin) v1.6.21
+3. Install [Kscript](https://github.com/holgerbrandl/kscript#installation) v4.0.3
+4. Run `cd scripts` to get into the scripts directory
+5. Run `kscript new_project.kts` to create a new project with the following arguments:
```

Comment:
Why are there these changes @Tuubz ? Isn't that we merged them to develop already? 🤔

Reply:
@lydiasama I've updated `README.md`, please have another look 🙇 

I have moved `Kotlin` & `Kscript` prerequisites out of the set-up section